src/styles.css

```diff
@@ -507,4 +507,9 @@ span.tooltip {
 .content table td {
   padding: 0.25em 0.5em;
 }
+
+.steps-content {
```

Comment:
In wizard no affect?

Reply:
No. It is required to show tab navigator on wizard.